google/cloud/forseti/common/util/logger.py

```diff
@@ -21,11 +21,12 @@
 import logging.handlers
 import os
 
+from google.cloud.forseti import __version__
 
 DEFAULT_LOG_FMT = ('%(asctime)s %(levelname)s '
                    '%(name)s(%(funcName)s): %(message).1024s')
 
-SYSLOG_LOG_FMT = ('%(levelname)s [forseti-security] '
+SYSLOG_LOG_FMT = ('%(levelname)s [forseti-security] %(__version__)s '
```

Comment:
We said we would put `[ ]` around the version.
`[forseti-security] [v2.3.0]`

Reply:
Consider it done.